cloudferrylib/os/identity/keystone.py

```diff
@@ -475,6 +489,35 @@ def _get_user_tenants_roles(self, tenant_list=None, user_list=None):
         if user_list is None:
             user_list = []
         user_tenants_roles = {}
+        if self.config.identity.mode == API_MODE:
+            user_tenants_roles = self.__get_user_tenants_roles_by_api(tenant_list,
+                                                                      user_list)
+        elif self.config.identity.mode == SQL_MODE:
+            user_tenants_roles = self.__get_user_tenants_roles_by_db(tenant_list,
+                                                                     user_list)
+        return user_tenants_roles
+
+    def __get_user_tenants_roles_by_db(self, tenant_list, user_list):
```

Comment:
I don't think there is any point in having double underscores here, single underscore is enough to mark method as private according to pep8. See [this](http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python) for more information.